liqd_product/assets/scss/components/_alerts.scss

```diff
@@ -0,0 +1,62 @@
+$messages-margin-bottom: 25px;
+
+.messages {
+    margin: 0 0 $messages-margin-bottom;
+    padding: 0;
+
+    li {
+        display: block;
+    }
+
+    + .messages {
+        margin-top: -$messages-margin-bottom;
+    }
+}
+
+.alert {
+    margin: 0;
+    padding: $padding * 1.2 0;
+    background-color: lighten($brand-info, 40);
```

Comment:
I do like that we always use body-text-color and the lightened colours as background and I know that you chcecked that it's a11y, buuuut does it look that different if we use contrast-stretch to make sure it still works if someone changes the colours in the variables.scss